src/header-generator.js

```diff
@@ -0,0 +1,310 @@
+const fs = require('fs');
+const path = require('path');
+const { BayesianNetwork } = require('bayesian-network');
+const { default: ow } = require('ow');
+
+const headerNetworkDefinitionPath = path.join(__dirname, './data_files/header-network-definition.json');
+const inputNetworkDefinitionPath = path.join(__dirname, './data_files/input-network-definition.json');
+const browserHelperFilePath = path.join(__dirname, './data_files/browser-helper-file.json');
+const headersOrderFilePath = path.join(__dirname, './data_files/headers-order.json');
+
+const browserHttpNodeName = '*BROWSER_HTTP';
+const operatingSystemNodeName = '*OPERATING_SYSTEM';
+const deviceNodeName = '*DEVICE';
+const missingValueDatasetToken = '*MISSING_VALUE*';
```

Comment:
I would expect those to be in CAPS as they are top level constants. I was looking for those variables in scope of the functions where they're used.

Reply:
Btw, why do all of them except `*MISSING_VALUE*` have a single asterisk? Is that a typo?

Reply:
As to the constants, I would have made them all caps, but I think checked one of our repositories and top level constants were written there this way, which is why I did it like this. Will try to remember which one it was and will make this ALL_CAPS.
The one vs two asterisk difference is actually there precisely to show that the constants are different things. The upper three are node names in the underlying bayesian network, the fourth is just a token representing empty value. I have added a space between them to reinforce the point.